resource_dispatcher/src/server.py

```diff
@@ -77,7 +81,7 @@ def do_POST(self):  # noqa: N802
 
 def generate_manifests(manifest_folder: str, namespace: str) -> list[dict]:
     """For each file in templates_folder generate a yaml with populated context."""
-    manifest_files = glob.glob(f"{manifest_folder}/*.yaml")
+    manifest_files = glob.glob(f"{manifest_folder}/*/*.yaml")
```

Comment:
I don't know glob patterns well, but I think this would work?
```suggestion
    manifest_files = glob.glob(f"{manifest_folder}/**/*.yaml")
```
That way you don't need to care how the user nests the files

I'm not 100% sure that catches files at the top level of `manifest_folder` though, you'd have to check